### PR TITLE
[plugin.onedrive] 2.0.0

### PR DESCRIPTION
**Pull request by cguZZman**
### Description
* Remake using script.module.clouddrive.common module
* Fix OneDrive for Business functionality
* **A heads up, just in case: Needs to be published together/after the new clouddrive module is published. PR: xbmc/repo-scripts#659**

### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

### CODE REVIEW

**Comment by Rechi** [comment]
@cguZZman you aren't allowed to submit more then one addon in a single PR. You have to open a second PR for `plugin.googledrive` and remove the second commit from here.

**Comment by cguZZman** [comment]
Thanks @Rechi. Yea, I committed it by mistake. But it's removed now.

**Comment by cguZZman** [comment]
Hey guys. Hope you can get some time to review the addon. The dependency got merged.

File: plugin.onedrive/addon.py [hunk 1]
```diff
@@ -0,0 +1,210 @@
+#-------------------------------------------------------------------------------
```
Comment (enen92) [inline plugin.onedrive/addon.py]: same comment as the google drive addon please consider to keep this file to a minimum in the future

Comment (cguZZman) [reply plugin.onedrive/addon.py]: will do

**Pull request merged**